### DISCUSSION TITLE
How to make colour aware of oiio on a mac?

### DISCUSSION THREAD

**Discussion by ahemberger** [colour-science/colour#1000] [Q&A] [answered]
Hi;

I installed opemi,mageio using homewbrew on a mac - I can invoke oiiotool from the terminal to confirm it works. Yet, when I run any script that uses colour, I see this warning:

```
/Users/xxxxx/Library/Python/3.7/lib/python/site-packages/colour/utilities/verbose.py:237: ColourUsageWarning: "OpenImageIO" related API features are not available, switching to "Imageio"!
  warn(*args, **kwargs)
```

What more must I do to help these two buddies find one another?

**Comment by ahemberger** [comment, marked as answer]
I'm unsure how to remove this item, but I eventually figured this out: oiio can be installed via homebrew, which has its own preferred Python version, which in turn controls which version of oiio one gets. As of this writing, that's Python 3.9; oiio won't be 'seeable' by other python versions. As I use pyenv to manage versions, I was a bit confused about how to manage this. 

My way to solve it was to install python 3.9, install oiio and dependencies, then install colour-science (I don't think this order matters, it's just what I did). Versions harmonized, colour and oiio seem to now 'just work'. The downside is that I'm boxed into a python version I don't prefer, but that's more an issue for the maintainers of the oiio homebrew installation, and has nothing to do with colour.

**Reply by thomasmansencal** [reply]
So what I usually do is that I install OIIO with Homebrew and symlink it in the Poetry Virtual Environments that I create for the various projects.
